src/procedures/procedures.11tydata.yaml

```diff
@@ -565,10 +571,61 @@ procedureDisableMultiProtocol:
               <li>Follow the instructions on flashing the SkyConnect firmware.</li>
             </ul>      
           </li>
+
+              <li>Plug SkyConnect back into your device.</li>
+        </ol>
+    - title: Rename the <b>Zigbee Home Automation (ZHA)</b> integration
+      image: skyconnect-multiprotocol-disable-03.png
+      content: |
+        <ul>
+        <li class="info">SkyConnect is now setup in the non-multiprotocol ZHA integration, although the title still includes <b>Multi-PAN</b>.</li>
+          
+        </ul>
+        <ol>
+        <li>Remove <b>Multi-PAN</b> from the name of the <b>Zigbee Home Automation (ZHA)</b> integration.</li>
+        <li>Select <b>Configure</b>.</li>
```

Comment:
For me, the button says "OK":

<img width="473" alt="image" src="https://user-images.githubusercontent.com/32534428/227599428-126e1158-0c3b-49e4-a360-7efead73cf16.png">

```suggestion
        <li>Select <b>OK</b>.</li>
```

I guess we could also remove this step, it seems straightforward?